client/post-editor/editor-page-slug/index.jsx

```diff
@@ -1,34 +1,24 @@
 /**
  * External dependencies
  */
-import React, { PropTypes } from 'react';
-import PureRenderMixin from 'react-pure-render/mixin';
-import pick from 'lodash/pick';
+import React, { PropTypes, Component } from 'react';
 
 /**
  * Internal Dependencies
  */
 import EditorSlug from 'post-editor/editor-slug';
 import Gridicon from 'components/gridicon';
 
-export default React.createClass( {
-	displayName: 'PostEditorPageSlug',
-
-	mixins: [ PureRenderMixin ],
-
-	propTypes: {
-		path: PropTypes.string,
-		slug: PropTypes.string
-	},
+export default class PostEditorPageSlug extends Component {
```

Comment:
Ah good catch - I have imported `PureComponent` elsewhere so not sure why I didn't do so here.